hack/update-openapigen.sh

```diff
@@ -26,8 +26,8 @@ fi
 go run vendor/k8s.io/code-generator/cmd/openapi-gen/main.go --input-dirs github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha2,knative.dev/pkg/apis --output-package github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha2/ --go-header-file hack/boilerplate.go.txt
 
 # Workaroud error "spec redeclared in this block" in unit test.
-sed -i 's%spec "github.com/go-openapi/spec"%openapispec "github.com/go-openapi/spec"%g' pkg/apis/serving/v1alpha2/openapi_generated.go
-sed -i 's/spec\./openapispec\./g' pkg/apis/serving/v1alpha2/openapi_generated.go
+sed -i '' -e 's%spec "github.com/go-openapi/spec"%openapispec "github.com/go-openapi/spec"%g' pkg/apis/serving/v1alpha2/openapi_generated.go
+sed -i '' -e 's/spec\./openapispec\./g' pkg/apis/serving/v1alpha2/openapi_generated.go
```

Comment:
you can delete this walkaround if you move these variables(https://github.com/kubeflow/kfserving/blob/master/pkg/apis/serving/v1alpha2/kfservice_framework_tensorrt_test.go#L41)  inside the function, it should not be declared at package level which causes conflicts.

Reply:
Thanks for your heads up! I confirm the fix passes all unit tests on my local machine. I'll push the commit and rename this PR.